howso/engine/trainee.py

```diff
@@ -3883,8 +3891,8 @@ def get_mda_matrix(
             be processed sequentially.
 
             Default Methods:
-            - 'relative': normalizes each row by dividing each value by the maximum absolute value in the row.
-            - 'fractional': normalizes each row by dividing each value by the sum of absolute values in the row.
+            - relative=normalizes each row by dividing each value by the maximum absolute value in the row.
+            - fractional=normalizes each row by dividing each value by the sum of absolute values in the row.
```

Comment:
Another instance of erroneous docstring changes.

Reply:
Addressed by: ae85d46